io.catenax.battery.battery_pass/3.0.1/BatteryPass.ttl

```diff
@@ -0,0 +1,920 @@
+    bamm:dataType :MaterialNameAndWeightAndPercentageMassFractionEntity.
+:AnodeActiveMaterialsCharacteristic a bamm-c:Set;
+    bamm:name "AnodeActiveMaterialsCharacteristic";
+    bamm:preferredName "anode active materials"@en;
+    bamm:description "The total amount of valuable materials contained in Anode: graphite"@en;
+    bamm:dataType :MaterialNameAndWeightAndPercentageMassFractionEntity.
+:CathodeCompositionOtherCharacteristic a bamm-c:Set;
+    bamm:name "CathodeCompositionOtherCharacteristic";
+    bamm:preferredName "cathode composition other"@en;
+    bamm:description "The composition or materials contained in the Cathode is describing a regulatory requirement."@en;
+    bamm:dataType :MaterialNameAndWeightAndPercentageMassFractionEntity.
+:AnodeCompositionOtherCharacteristic a bamm-c:Set;
+    bamm:name "AnodeCompositionOtherCharacteristic";
+    bamm:preferredName "anode composition other"@en;
+    bamm:description "The composition or materials contained in the anode is describing a regulatory requirement."@en;
+    bamm:dataType :MaterialNameAndWeightAndPercentageMassFractionEntity.
+:ElectrolyteCompositionCharacteristic a bamm-c:Set;
+    bamm:name "ElectrolyteCompositionCharacteristic";
+    bamm:preferredName "electrolyte composition"@en;
+    bamm:description "List of materials contained in the electrolyte is describing a regulatory requirement."@en;
+    bamm:see <https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:52020DC0474>;
+    bamm:dataType :MaterialNameAndWeightAndPercentageMassFractionEntity.
+:ComponentsCharacteristic a bamm:Characteristic;
+    bamm:name "ComponentsCharacteristic";
+    bamm:preferredName "components"@en;
+    bamm:description "Characteristics of the batterie's comoponents."@en;
+    bamm:dataType :CompomenentsEntity.
+:CompositionOfBatteryCharacteristic a bamm-c:Set;
+    bamm:name "CompositionOfBatteryCharacteristic";
+    bamm:preferredName "composition of battery"@en;
+    bamm:description "Full composition of battery is describing a business requirement."@en;
+    bamm:dataType :MaterialNameAndWeightAndPercentageMassFractionEntity.
+:CriticalRawMaterialsCharacteristic a bamm:Characteristic;
+    bamm:name "CriticalRawMaterialsCharacteristic";
+    bamm:preferredName "critical raw materials"@en;
+    bamm:description "List of critical raw materials (CRM) as specified by EU in a battery is describing a regulatory requirement."@en;
+    bamm:dataType xsd:string.
+:TemperatureRangeCharacterisitic a bamm-c:Measurement;
+    bamm:name "TemperatureRangeCharacterisitic";
+    bamm:preferredName "temperature range"@en;
+    bamm:description "The lower range of temperature the battery can withstand when not in use is describing a regulatory requirement."@en;
+    bamm:dataType xsd:decimal;
+    bamm-c:unit unit:degreeCelsius.
```

Comment:
Are temperature ranges not measured in Kelvin?

Reply:
domain experts request to have it in celsius